src/main/java/MainWindow.java

```diff
@@ -0,0 +1,76 @@
+import duke.DialogBox;
+import duke.Duke;
+
+import duke.MainUi;
```

Comment:
It's good that you never use wildcard imports but it would be better if you grouped the imports by packages.

Reply:
Not sure when that line break got in. But good observation.